vendor/minol-zenner/edc-communication-module-profile.yaml

```diff
@@ -0,0 +1,21 @@
+# Vendor profile ID, can be freely issued by the vendor
+# This vendor profile ID is also used on the QR code for LoRaWAN devices, see
+# https://lora-alliance.org/sites/default/files/2020-10/LoRa_Alliance_Vendor_ID_for_QR_Code.pdf
+vendorProfileID: 474
```

Comment:
I want to quickly notify you on changing this. The vendorProfileID is different from the VendorID. This vendorProfileID should be an incremental value, used for identifying devices within a vendor. So change it to something along the lines of 1, 101, etc.

I shall have that text changed to better reflect what it's supposed to be.

Reply:
Thanks for the clarification. I changed the vendorProfileId to 1